kvm/v12.3.2/README.md

```diff
@@ -0,0 +1,25 @@
+# :zap: Giant Swarm Release v12.3.2 for KVM :zap:
+
+**Nodes will be rolled during upgrade to this version.**
+
+This patch release prevents an issue with QPS limits introduced by DockerHub.
+
+**Note before upgrade:**
+
+Please contact your Solution Engineer before upgrading. The upgrade is automated. However, it includes a data migration from Helm 2 release configmaps to Helm 3 release secrets, there are some pre-upgrade checks and we recommend monitoring the upgrade to ensure safety.
```

Comment:
Is this still relevant?

Reply:
I assumed it was related to customers upgrading from KVM `v12.x.x`. I can drop this section and the note to SREs if you think this is irrelevant.

Reply:
Makes sense. I don't mind keeping it.